website/templates/dissemin/tos.html

```diff
@@ -118,5 +118,81 @@ <h3 id="CNIL">Déclaration CNIL</h3>
             </p>
         </div>
     </div>
+    <div class="card-mb-3">
+        <div class="card-header bg-light-blue text-white">
+            <h2>Privacy Policy</h2>
+        </div>
+        <div class="card-body">
+            <p>This privacy policy describes the private data usage and processing. It mainly addresses the usage when logged in via Federated Login (Shibboleth). The data private data usage and processing when logged via another authentication method may vary in detail.</p>
+            <h3>Name of the Service</h3>
+            <p>Dissemin</p>
+
+            <h3>Description of the Service</h3>
+            <p>Free and open service to help researchers deposit their articles in open repositories with respect to self-archiving policies of the publisher and in align with guidelines of employers or funders. The service is supported by CCSD (HAL), CERN (Zenodo) and the universities of Braunschweig, Darmstadt and Stuttgart.
+            </p>
+            <h3>Data Controller and Contact Person</h3>
+            <p>
+                Association CAPSH<br />
+                34 rue de la Chanaise<br />
+                71250 Cluny<br />
+                France
+            </p>
+
+            <h3>Jurisdiction</h3>
+            <p>The provisions of French law apply to this agreement. Place of jurisdiction is Cluny.</p>
+
+            <h3>Personal Data Processed</h3>
+            <p>In case of login via Federation Login (Shibboleth), following personal data is processed:</p>
+            <ul>
+                <li>
+                    Data requested as required from users home organization:
+                    <ul>
+                        <li>SAML persistent identifier, unique identifier to identify the user (attribute: eduPersonTargetedID)</li>
+                        <li>users name (attributes: givenName, sn</li>
+                    </ul>
+                </li>
+                <li>
+                    Data requested optional from users home organization:
+                    <ul>
+                        <li>users email (attribute: mail)</li>
+                        <li>users ORCID (attribute: eduPersonOrcid</li>
+                    </ul>
+                </li>
+                <li>
+                    Data gathered from the user:
+                    <ul>
+                        <li>In case the home organisation does not provide some of the requested attributes, the user might be requested to provide the missing information in order to use certain functionality, or otherwise this functionality may not be available to the user.</li>
```

Comment:
```suggestion
                        <li>If the home organisation does not provide some of the requested attributes, the user might be requested to provide the missing information in order to use certain functionality, or otherwise this functionality may not be available to the user.</li>
```